xml/bp_troubleshooting_cephfs.xml

```diff
@@ -271,4 +271,382 @@ sudo zypper in ceph-common
      </para>
    </sect2>
  </sect1>
+ <sect1 xml:id="bp-troubleshooting-old-kernel-clients">
+   <title>Using legacy kernel clients</title>
```

Comment:
I think we should explain very briefly what a "kernel client" is. (I did not know.)

1 paragraph would be enough.

Reply:
Right, I'll try to rephrase the initial paragraphs to make it a bit more clear.  However, my opinion is that a troubleshooting guide necessarily needs to assume some prior knowledge -- defining what a "kernel client" is could be something for another guide.

(Thanks a lot for your review, BTW!)

Reply:
> Right, I'll try to rephrase the initial paragraphs to make it a bit more clear. However, my opinion is that a troubleshooting guide necessarily needs to assume some prior knowledge -- defining what a "kernel client" is could be something for another guide.

I take your point, yes -- although we could afford a sentence. :-) The question is, though, whether the kernel client is tied to the kernel version, and if the kernel version is tied to the client machine's OS version.

(For example, in Ubuntu releases "hardware enablement stacks" for its Long Term Support (LTS) releases. These contain a kernel, X server etc. backported from the current short-term support release. This means that it is possible to stay with a given OS release (say, 18.04) and nonetheless install a newer kernel. 

> (Thanks a lot for your review, BTW!)

You're welcome. It's what I am here for. :-)